tools/install-source/Program.cs

```diff
@@ -207,10 +207,7 @@ public static int Main (string[] arguments)
 			}
 			if (verbose)
 				Console.WriteLine ($"Original source is {src}");
-			string fixedSource = src;
-
-			if (!File.Exists (fixedSource))
-				fixedSource = mangler.GetSourcePath (src);
+			var fixedSource = mangler.GetSourcePath (src);
```

Comment:
out of curiosity, why is this needed now ?

Reply:
It's a revert of this fix, which turned out to be wrong: cf126061a98b81300ded3974b60817af0efcbff4.